app/src/pages/inside/dashboardItemPage/widgetsGrid/emptyWidgetGrid/emptyWidgetGrid.jsx

```diff
@@ -42,33 +39,21 @@ const messages = defineMessages({
 });
 
 @injectIntl
-@connect((state) => ({
-  userInfo: userInfoSelector(state),
-  projectRole: activeProjectRoleSelector(state),
-}))
 export class EmptyWidgetGrid extends Component {
   static propTypes = {
     intl: PropTypes.object.isRequired,
     action: PropTypes.func,
     isDisable: PropTypes.bool,
-    dashboard: PropTypes.object,
-    userInfo: PropTypes.object,
-    projectRole: PropTypes.string,
   };
 
   static defaultProps = {
     action: () => {},
     isDisable: false,
-    dashboard: {},
-    userInfo: {},
-    projectRole: '',
   };
 
   render() {
-    const { action, intl, isDisable, dashboard, userInfo, projectRole } = this.props;
-    const isAddWidgetEnabled =
-      canAddWidget(userInfo.userRole, projectRole, dashboard.owner === userInfo.userId) &&
-      !isDisable;
+    const { action, intl, isDisable } = this.props;
+    const isAddWidgetEnabled = !isDisable;
```

Comment:
Now it is not necessary to create additional variable for that, you can use `!isDisable` direcrtly.